nim/Dockerfile

```diff
@@ -25,6 +25,9 @@ RUN nim c {{#build_opts}} {{{.}}} {{/build_opts}} \
 
 FROM alpine
 
+RUN apk --no-cache add \
```

Comment:
is this not specific to `basolato` ?

Reply:
@waghanza 
This is specific for `basolato`. However due to multi stage building, it is necessary to run.
Other frameworks arre minimal and didn't use `pcre`